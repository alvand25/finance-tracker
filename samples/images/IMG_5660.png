version https://git-lfs.github.com/spec/v1
oid sha256:564c4855b3afcea5d203bce3646110d66ff310cc398ebb7cd022f232ce7519c8
size 13028131
