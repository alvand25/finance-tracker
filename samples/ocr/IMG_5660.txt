http://www. hmart,com :
48-18 Northern Bivd
Long Is1and City, NY. 11101
TEL (718) 489-8580 ,
Your Cashier woo YUN JUNG JANG \
ROTI55ERIE CHICKEN 5.99 B
j Regu1ar Price 7.59 , You saved 2.00 '
JIN RAMEN HILB BOX 16,99 F
Regu1ar Price 22,99 , You saved 6.00 \
5NOW ICE/COOKI&CRM 8.99 F ,
MINI CUCUMBER5 1.49 F .
\ Regu1ar Price 2.99 , You saved 1.50
MRYM LATTE 2,99 F
ICECAFE AMERICANG 1.99 B
\ MA5AMAN CURRY PA5T 1.99 F a
5RKK GYOZA- BEEF 5.99 F i
; Regu1ar Price 7.99 , Yau saved 2,00 \
BGR 5AMANCO CHOCO 4.99 F \
Regu1ar Price 7.99 , You saved 3.30
1.86 Ib 0 2.99 /Ib \
WT GRN 5EEDLE55 GRAPE 5.56 F
Regu1ar Price 3,99 , You saved 1,86
GROCERY 3.75 F
PAPER CARRYOUT BAG 005 8
PAPER CARRYOUT BAG 005 B
TAK 0.72 .
ven BALANCE 61.64