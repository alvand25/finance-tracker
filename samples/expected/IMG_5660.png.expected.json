{
    "store": "Trader_Joes",
    "items": [
        {
            "description": "SPICY CHAI TEA LATTE MIX",
            "price": 4.99
        },
        {
            "description": "ORG ELOTE CORN CHIPS",
            "price": 2.99
        },
        {
            "description": "CREAM CHEESE BRICK",
            "price": 3.98
        }
    ],
    "subtotal": 11.96,
    "tax": 0.0,
    "total": 11.96,
    "payment_method": "MasterCard",
    "timestamp": "2024-04-14T11:35:00",
    "check_items_exactly": true
}